src/uwtools/tests/test_schemas.py

```diff
@@ -594,6 +601,52 @@ def test_schema_make_hgrid_run_dir(make_hgrid_prop):
     assert "88 is not of type 'string'" in errors(88)
 
 
+# make_solo_mosaic
+
+
+def test_schema_make_solo_mosaic():
+    config = {
+        "config": {"dir": "path/to/dir", "num_tiles": 1},
+        "execution": {"executable": "make_solo_mosaic"},
+        "run_dir": "/tmp",
+    }
+    errors = schema_validator("make-solo-mosaic", "properties", "make_solo_mosaic")
+    # Basic correctness:
+    assert not errors(config)
+    # All top-level keys are required:
+    for key in ("config", "execution", "run_dir"):
+        assert f"'{key}' is a required property" in errors(with_del(config, key))
+    # Additional top-level keys are not allowed:
+    assert "Additional properties are not allowed" in errors({**config, "foo": "bar"})
+
+
+def test_schema_make_solo_mosaic_config():
+    # Get errors function from schema_validator
+    errors = schema_validator(
+        "make-solo-mosaic", "properties", "make_solo_mosaic", "properties", "config"
+    )
```

Comment:
Can we receive the `make_solo_mosaic_prop` fixture here and then
```
errors = make_solo_mosaic_prop("config")
```
here. IIRC that's the pattern in the tests for the other driver configs, except for `test_schema_make_solo_mosaic()` and its analogs for other drivers, where we need to start a level higher in the schema.

Reply:
Updated. Thank you for all of your feedback on this one.